partner_credit_limit/models/partner.py

```diff
@@ -92,3 +95,23 @@ def get_allowed_sale(self):
         for partner in self:
             partner.allowed_sale = not partner.credit_overloaded and \
                 not partner.overdue_credit
+
+    def _check_limit_credit(self):
+        """Method that will return a message when the partner has late payments and/or has exceeded the credit limit.
+        """
+        self.ensure_one()
+        if self.allowed_sale:
+            return False
+        credit_overloaded = self.credit_overloaded
+        overdue_credit = self.overdue_credit
+        msg = _('Can not confirm the %s because the Partner:\n') % (self.env.context.get('model_name', _('record')))
```

Comment:
instead of passing these parameters by context, why not simply passing them as standard method parameters?

Reply:
Done